client/components/draft_input/components/input_caption.coffee

```diff
@@ -0,0 +1,212 @@
+React = require 'react'
+window.global = window
+window.process = {env: {NODE_ENV: 'development'}}
```

Comment:
Weirdly Draft need's these Node variables, but that's a good point—we may want to use [envify](https://github.com/hughsk/envify) to inject this conditionally.